site/docs/features/index.md

```diff
@@ -1,107 +1,327 @@
-some other tool, each mutation operation you do will add or delete one or more files from 
-the definition of your table. Nessies keeps tracks of which files are related to each 
-of your tables at every point in time and then allows you to recall those as needed.
+| Attribute | Meaning
+| --- | ---
+| commit timestamp | the timestamp when the commit has been recorded in Nessie
+| committer | the one (human user, system id) that actually recorded the change in Nessie
+| author timestamp | the timestamp when a change has been implemented (can be different from the commit timestamp)
+| author | the one (human user, system id) that authored the change. can be different, if someone else actually commits the change to Nessie
+| summary | one-line meaningful summary of the changes
+| description | potentially long description of the changes
+| ... | there are potentially way more attributes, just too many to mention here
 
-## Scale & Performance
+## Garbage collection
 
-Nessie is built for very large data warehouses. Nessie [supports](../develop/kernel.md) 
-millions of tables and 1000s of commits/second. Because Nessie builds on top of Iceberg 
-and Delta Lake, each table can have millions of files. As such, Nessie can support 
-data warehouses several magnitudes larger than the largest in the world today. This 
-is possible in large part due to the separation of transaction management (Nessie) from 
-table metadata management (Iceberg and Delta Lake).
+Data lakes contain a lot of data. The amount of data has a direct relation to
+the cost of ownership of a data lake. Keeping all data forever is probably going
+to be just too expensive, practically not useful and can also collide with data
+privacy regulations (for example GDPR or CCPA).
 
-## Technology 
-Nessie can be [deployed in multiple ways](../try) and is composed primarily of the Nessie service, 
-which exposes a set of [REST APIs](../develop/rest.md) and a simple browse UI. This service works with multiple
-libraries to expose Nessie version control capabilities to common data management technologies.
+Nessie keeps track of unused data files and collects the garbage for you.
 
-Nessie was built as a Cloud native technology and is designed to be highly scalable, 
-[performant](../develop/kernel.md) and resilient. Built 
-on Java and leveraging [Quarkus](https://quarkus.io/), it is compiled to a GraalVM native image 
-that starts in less than 20ms. This makes Nessie work very well in docker and FaaS environments. 
-Nessie has a pluggable storage backend and comes pre-packaged with support for DynamoDB and local 
-storage.
+## Terms summary
 
-## License and Governance
-Nessie is Apache-Licensed and built in an open source, consensus-driven GitHub community. 
-Nessie was originally conceived and built by engineers at [Dremio](http://dremio.com).
+| Term | Meaning in Nessie
+| --- | ---
+| Commit | An atomic change to a set of data files.
```

Comment:
I'd suggest using this format of the second column starting with a capital letter and ending with a full stop in the "Attribute | Meaning" table above too.

Reply:
Yup, applied to the the above table.